test/extended/operators/olm.go

```diff
@@ -70,4 +70,30 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			}
 		})
 	}
+
+	// OCP-20981, [BZ 1626434]The olm/catalog binary should output the exact version info
+	// author: jiazha@redhat.com
+	g.It("olm version should contain the source commit id", func() {
+		catalogPodName, err := oc.AsAdmin().Run("get").Args("-n", "openshift-operator-lifecycle-manager", "pods", "-l", "app=catalog-operator", "-o=jsonpath={.items[0].metadata.name}").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		e2e.Logf("catalog-operator pod name:%s", catalogPodName)
+
+		g.By("get olm version from the catalog-operator pod")
+		oc.SetNamespace("openshift-operator-lifecycle-manager")
+		commands := []string{"exec", catalogPodName, "--", "olm", "--version"}
+		olmVersion, err := oc.AsAdmin().Run(commands...).Args().OutputToFile("version.json")
```

Comment:
nit: why `version.json`? The contents aren't json, right?

Reply:
Yes, I just want to save it to a file. Maybe I should use the YAML, thanks!

Reply:
Done.